### PR TITLE
feat(FE): add production build & serve local script

### LINKED ISSUE #498: run production build in locally server
## Bug description

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  

run production build in local server

## Expected behavior
write a script while will run the production build locally (production build=>`/build`)

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
Signed-off-by: Siddhant Khare <Siddhantkhare2694@gmail.com>

fixes #498

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
 	"scripts": {
 		"dev": "NODE_ENV=development webpack serve --progress",
 		"build": "webpack --config=webpack.config.prod.js --progress",
+		"prod": "yarn build && npx serve ./build",
```
Comment (pal-sig) [inline frontend/package.json:9]: can we remove npx here

File: frontend/package.json [hunk 2]
```diff
@@ -58,6 +59,7 @@
 		"react-vis": "^1.11.7",
 		"redux": "^4.0.5",
 		"redux-thunk": "^2.3.0",
+		"serve": "^13.0.2",
```
Comment (pal-sig) [inline frontend/package.json:62]: can we use this as devDependencies here

**Review by pal-sig — CHANGES_REQUESTED** [review]

**Comment by pal-sig** [comment]
also can you re-route all request to `/index.html` seems it is reflecting 404 when we refresh

**Comment by pal-sig** [comment]
@Siddhant-K-code still the same issue

**Comment by Siddhant-K-code** [comment]
are we facing the same issue on `dev` version?
if this is the issue with `build` folder path issues, then we have to make some changes. But I don't think it is because of it.

I think we have to handle some routes for `/`

**Comment by palash-signoz** [comment]
> are we facing the same issue on `dev` version? if this is the issue with `build` folder path issues, then we have to make some changes. But I don't think it is because of it.
> 
> I think we have to handle some routes for `/`

Yes @Siddhant-K-code

**Comment by palash-signoz** [comment]
You can use `https://github.com/vercel/serve-handler#rewrites-array`

**Comment by palashgdev** [comment]
Closing due to inactivity

**Pull request closed without merging**